letsencrypt-aws.py

```diff
@@ -1,154 +1,266 @@
 import datetime
```

Comment:
Source file is pretty large with a lot of different concerns going on (command line parsing, multiple subcommands, etc...).  Might be due for breaking up into some different modules.